book/stages/README.md

```diff
@@ -0,0 +1,80 @@
+# Stages
+
+The `stages` lib plays a central role in syncing the node, maintaining state, updating the database and more. The stages involved in the Reth pipeline are the `HeaderStage`, `BodyStage`, `SendersStage`, and `ExecutionStage`. Each of these stages are queued up and stored within the Reth pipeline.
```

Comment:
pls add a comment that this list is nonexhaustive, more will be added shortly

Reply:
Sounds good, just added a note about this.